tasks/appserver_config.yml

```
@@ -146,4 +146,11 @@
   become: true
   become_user: "{{ data_server_user }}"
   no_log: true  # Ensure passwords aren't logged if this fails.
-
+  
+- name: Increase user level soft/hard FD limits
+  blockinfile:
+    path: /etc/security/limits.conf
+    block: |
+          '*     soft nofile 999999'
```

You don't need the quotes, and I probably wouldn't set it globally. I'd suggest something like this and my next suggested change, instead:
```suggestion
      {{ data_server_user }}     soft nofile 999999
```

Please verify I got that right, and check it in TEST, though -- I haven't tested this.